src/main.js

```diff
@@ -222,6 +224,7 @@ class MiniGraphCard extends LitElement {
         ?fill=${config.show.graph && config.show.fill}
         ?points=${config.show.points === 'hover'}
         ?labels=${config.show.labels === 'hover'}
+        ?labels-secondary=${config.show.labels_secondary === 'hover'}
```

Comment:
Need to se the default of `hover` in `const.js` `DEFAULT_SHOW`, similar to the default labels.

Reply:
Done